src/admin/components/ResourceTabs/InformationTab/components/ReportInformationAside/HighLevelMetrics.tsx

```diff
@@ -26,45 +27,49 @@ const HighLevelMetics: FC = () => {
 
       <Box paddingX={3.75} paddingY={2}>
         <Stack gap={3}>
-          <ContextCondition frameworksHide={[Framework.TF]}>
-            <Labeled label="Total Number Of Workdays Created" sx={inlineLabelSx}>
-              <NumberField source={record.workdays_total ? "workdays_total" : "total_workdays_count"} emptyText="0" />
-            </Labeled>
-            <When condition={workdaysType != null}>
-              <Labeled label={`Total Number Of Paid ${workdaysType} Workdays Created`} sx={inlineLabelSx}>
-                <NumberField source="workdays_paid" emptyText="0" />
+          <When condition={!isSiteReport}>
+            <ContextCondition frameworksHide={[Framework.TF, Framework.TF_LANDSCAPES]}>
```

Comment:
Should this include TF Enterprises? Assuming yes, this should instead be `frameworksHide={ALL_TF}` (ALL_TF is an export from the same file as the `Framework` enum.